locales/de.json

```diff
@@ -1,574 +1,911 @@
+      "fail": "Fokussierbarer Inhalt sollte deaktiviert oder vom DOM entfernt werden."
+    },
+    "focusable-element": {
+      "pass": "Element ist fokussierbar.",
+      "fail": "Element sollte fokussierbar sein."
+    },
+    "focusable-modal-open": {
+      "pass": "Keine fokussierbaren Elemente während ein Model offen ist.",
+      "incomplete": "Üerprüfe ob Elemente fokussierbar während des derzeitigen Status sind."
+    },
+    "focusable-no-name": {
+      "pass": "",
+      "fail": "Das Element befindet sich in der Tabreihenfolge und enthält keinen erreichbaren Text.",
+      "incomplete": "Es ist nicht möglich herauszufinden ob das Element einen zugänglichen Namen besitzt."
+    },
+    "focusable-not-tabbable": {
+      "pass": "Keine fokussierbaren Elemente innerhalb des Elements.",
+      "fail": "Fokussierbare Elemente sollten mit tabindex='-1' versehen oder vom DOM entfernt werden."
+    },
+    "landmark-is-top-level": {
+      "pass": "",
+      "fail": "Die ${data.role} landmark befindet sich innerhalb einer anderen landmark."
+    },
+    "page-has-heading-one": {
+      "pass": "",
+      "fail": "Die Seite muss eine Überschrift erster Ebene besitzen."
+    },
+    "page-has-main": {
+      "pass": "",
+      "fail": "Die Seite muss eine main landmark besitzen."
+    },
+    "page-no-duplicate-banner": {
+      "pass": "",
+      "fail": "Das Dokument besitzt mehr als eine banner landmark."
+    },
+    "page-no-duplicate-contentinfo": {
+      "pass": "",
+      "fail": "Das Dokument besitzt mehr als eine contentinfo landmark."
+    },
+    "page-no-duplicate-main": {
+      "pass": "",
+      "fail": "Das Dokument besitzt mehr als eine main landmark."
+    },
+    "tabindex": {
+      "pass": "",
+      "fail": "Das Element besitzt einen tabindex-Attributwert größer als 0."
+    },
+    "alt-space-value": {
+      "pass": "Element hat ein valides alt Attribut.",
+      "fail": "Element hat ein alt Attribut, welches ausschließlich Leerzeichen beeinhaltet, welche jedoch nicht durch Screenreader ignoriert werden."
+    },
+    "duplicate-img-label": {
+      "pass": "",
+      "fail": "Das Element besitzt ein <img>-Element mit Alternativtext, der vorhandenen Text wiederholt."
+    },
+    "explicit-label": {
+      "pass": "",
+      "fail": "Das <form>-Element besitzt kein explizites <label>.",
+      "incomplete": "Es ist nicht möglich herauszufinden ob das <form> Element ein explizites <label> besitzt."
+    },
+    "help-same-as-label": {
+      "pass": "",
+      "fail": "Der Hilfstext (angegeben durch ein title- oder aria-describedby-Attribut) wiederholt den Label-Text."
+    },
+    "hidden-explicit-label": {
+      "pass": "Das <form> Element besitzt ein sichtbares explizites <label>.",
+      "fail": "Das <form> Element besitzt ein <label>, welches nicht sichtbar ist.",
+      "incomplete": "Nicht möglich herauszufinden ob <form> Element ein nicht sichtbares <label> besitzt."
```

Comment:
change to `Nicht möglich herauszufinden ob <form> Element ein sichtbares <label> besitzt.`